app/src/main/res/values-de/strings.xml

```diff
@@ -201,6 +212,12 @@
 
     <string name="settings_title_ask_delete">Vor Löschen bestätigen</string>
 
+    <string name="settings_title_storage">Speichermedium auswählen</string>
+    <string name="settings_title_storage_internal">Interner Speicher</string>
+    <string name="settings_title_storage_external">SD-Karte</string>
+    <string name="settings_summary_storage">%s</string>
```

Comment:
You format a string by just returning the same string? :D

Reply:
:flushed: